cdc/sink/black_hole.go

```diff
@@ -68,9 +68,9 @@ func (b *blackHoleSink) EmitCheckpointTs(ctx context.Context, ts uint64) error {
 	return nil
 }
 
-func (b *blackHoleSink) EmitDDLEvent(ctx context.Context, ddl *model.DDLEvent) error {
+func (b *blackHoleSink) EmitDDLEvent(ctx context.Context, ddl *model.DDLEvent) (EmitEventResult, error) {
```

Comment:
We can avoid changing the Sink interface by using a special error to indicate that the DDL is ignored

Reply:
Use special error is quite a hack way. Changing the export interface is not always a bad thing, but I was considering whether to put the result to each `EmitXXXEvent`, but this may decrease performance

Reply:
Ignoring can also be expressed as an error, I don't think it's hack

Reply:
addressed